### PR TITLE
Move Trento test to qesap.py

### PR DESCRIPTION
**Pull request by mpagot**
Move Trento test SAP deployment from build.sh to qesap.py. Minor qesapdeploy lib API change to not directly use PUBLICCLOUD settings.

- Verification run: http://openqaworker15.qa.suse.cz/tests/49008 http://openqaworker15.qa.suse.cz/tests/49518 http://openqaworker15.qa.suse.cz/tests/49519

### CODE REVIEW

File: lib/qesapdeployment.pm [hunk 1]
```diff
@@ -353,7 +354,8 @@ sub qesap_execute {
 
     my $verbose = $args{verbose} ? "--verbose" : "";
     my %paths = qesap_get_file_paths();
-    my $exec_log = "/tmp/qesap_exec_" . $args{cmd} . ".log.txt";
+    my $exec_log = "/tmp/qesap_exec_$args{cmd}_$args{cmd_options}.log.txt";
+    $exec_log =~ s/[-\s]+/_/g;
```
Comment (lpalovsky) [inline lib/qesapdeployment.pm:358]: Thanks for the log name fix !
And you even included the cmd option :+1:

**Review by lpalovsky — APPROVED** [review]
LGTM :+1:

File: lib/qesapdeployment.pm (hunk 1 above)
Comment (mpagot) [reply lib/qesapdeployment.pm:358]: It is about fixing the problem of
```
qesap_execute(cmd => 'terraform', verbose => 1, timeout => 1200);
```
and
```
qesap_execute(cmd => 'terraform', cmd_options => '-d', verbose => 1, timeout => 1200);
```
Producing a log file with the same name. 
My proposed solution works but it is not optimal for commands that are not using **cmd_options**, like http://openqaworker15.qa.suse.cz/tests/49519/logfile?filename=cluster_configure-qesap_exec_configure_.log.txt (do you see, the underscore at the end)

**Comment by jankohoutek** [comment]
LGTM

**Review by jankohoutek — COMMENTED** [review]
LGTM

**Pull request merged**